standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/minihms/AbstractMetaStoreService.java

```diff
@@ -99,7 +99,7 @@ public void start(Map<MetastoreConf.ConfVars, String> metastoreOverlay,
    * @return The client connected to this service
    * @throws MetaException if any Exception occurs during client configuration
    */
-  public IMetaStoreClient getClient() throws MetaException {
```

Comment:
nit: Is there any particular reason to change it to return `HiveMetaStoreClient`

Reply:
It allows access to methods like setProcessorCapabilities in the children tests. I did not see any indication the tests were specifically designed to test the IMetaStoreClient, so I felt it was safe to expose HiveMetaStoreClient to have better access to HMSClient methods.